include/mgba/feature/commandline.h

```diff
@@ -29,6 +37,9 @@ struct mArguments {
 	bool debugAtStart;
 	bool showHelp;
 	bool showVersion;
+
+	char hwExtensions;
+	char hwExtensionsFlags[HWEX_EXTENSIONS_COUNT];
```

Comment:
See above about configuration. Passing in via a -C flag is probably a better idea, at least for now.

Reply:
Also removed this.